cmd/tsidp/tsidp.go

```diff
@@ -520,6 +542,140 @@ type userInfo struct {
 	UserName string `json:"username"`
 }
 
+type capRule struct {
+	IncludeInUserInfo bool                   `json:"includeInUserInfo"`
+	ExtraClaims       map[string]interface{} `json:"extraClaims,omitempty"` // list of features peer is allowed to edit
```

Comment:
Aha! I understand what's happening here. The CI linter has an extra rule to expect `any` instead of `interface{}`. (We probably should use `gofmt -s` instead, which would do that)

Reply:
thank you for the 🕵🏻 work! I'll merge this and we can fast-follow with a fix to prevent another round of PR waiting.